src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class FindCommand extends Command{
+    private final String fullCommand;
+
+    public FindCommand(String fullCommand) {
+        this.fullCommand = fullCommand;
+    }
+
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) {
+        String keyword = fullCommand.substring(4).strip();
+        taskList.displayFind(keyword);
```

Comment:
Try to avoid magic number in this case, people reading "4" might not understand why. I believe that a split command based on spacebar notation will make people understand that you are trying to separate the first word from the rest of the command line